addons/website/views/snippets/snippets.xml

```diff
@@ -455,17 +455,19 @@
              t-att-data-selector="selector"
              t-att-data-exclude="exclude"
              t-att-data-target="target">
-            <we-select string="Parallax"
+            <we-select string="Scroll Effect"
                        class="o_we_sublevel_2"
-                       data-attribute-name="scrollBackgroundRatio"
+                       data-attribute-name="parallaxType"
                        data-attribute-default-value="0"
                        data-parallax-type-opt="true"
                        data-no-preview="true"
                        data-dependencies="bg_image_opt">
-                <we-button data-name="parallax_none_opt" data-select-data-attribute="0">None</we-button>
-                <we-button data-select-data-attribute="1">Fixed</we-button>
-                <we-button data-name="parallax_top_opt" data-select-data-attribute="1.5">Bottom to Top</we-button>
-                <we-button data-name="parallax_bottom_opt" data-select-data-attribute="-1.5">Top to Bottom</we-button>
+                <we-button data-name="parallax_none_opt" data-select-data-attribute="none">None</we-button>
+                <we-button data-select-data-attribute="fixed">Fixed</we-button>
+                <we-button data-name="parallax_top_opt" data-select-data-attribute="top">Parallax to Top</we-button>
+                <we-button data-name="parallax_bottom_opt" data-select-data-attribute="bottom">Parallax to Bottom</we-button>
```

Comment:
Your PR seems nice :+1: I think it can be even better if you don't save that "none", "top" and "bottom" information in the DOM. You should still define it like that but your JS code can make it so it does not land in the DOM if not necessary. The parallax animation itself does not seem to use it, so it's nice: old blocks in old databases will still work without us doing anything. However, the options in the editor will stop showing the right value because old blocks won't have the parallax-type data. What you could do instead is:
- Make sure the option method (`selectDataAttribute` here but TBH, you should rather implement a custom one, like `setParallaxType` it will be clearer) only sets the data-XXX value for the zoom options. For the others, it does as before: adding the right classes, etc.
- Adapt the `_computeWidgetState`: if there is a data-parallax-type, use it. If not rely on the speed value / classes to determine which of the other options it is.

This will be better as it will support previous DB without upgrade script / compat code etc but this is mainly also better from the new users point of view: their DOM will not be bigger for the same options that already existed (they don't need the data-parallax-type for the existing options). It is a small gain of course but 1+1+1+1+1... we try to avoid adding non-necessary data in the user DOM :+1:

Reply:
Nice hints! Thanks! So you're suggesting to keep the `data-select-data-attribute` as before (with the values: 0,1,-1.5 and 1.5) and add the _"type annotations"_ ("zoom_in" and "zoom_out") only for the zoom effects?

Reply:
> you should rather implement a custom one, like setParallaxType it will be clearer

I am suggesting that :point_up: It would be confusing at this point to use the `selectDataAttribute` generic method given all the conditional stuff we do (it already was dubious before your PR). Do kinda as you did but use a whole new method name, like `setParallaxType`.